### ISSUE TITLE
Add label

### ISSUE THREAD

**Issue by Priyanka-43** [ubsuny/CP1-24-HW1#10]
How to add label to new issue?

**Comment by ojha-aditya** [comment]
I believe @laserlab had mentioned something around we have limited permission before we start working on the repository and we would get permissions after some time on its own.

Although I also do not see an option yet.

**Comment by Priyanka-43** [comment]
Thankyou, for letting me know.

**Comment by laserlab** [comment]
@Priyanka-43 You should have gotten an invite to join the CP1 team. Then you should be able to make labels.

**Comment by Priyanka-43** [comment]
Yes, I am able to make label now. Thankyou.
